### PR TITLE
Add Workers (#55)

### PR DESCRIPTION
**Pull request by jbampton**

### CODE REVIEW

File: packages/john/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <!-- Twitter Meta Tags -->
+    <meta property="twitter:domain" content="john.onelang.workers.dev">
+    <meta property="twitter:url" content="https://john.onelang.workers.dev">
+    <meta name="twitter:title" content="The One Lang Magazine">
+    <meta name="twitter:description" content="Police Quest and the Forever One">
+  </head>
+  <body>
+    <h1>Hello John One!!</h1>
+  </body>
+</html>`
```
Comment (jbampton) [inline packages/john/index.js:27]:
```suggestion
</html>`;
```

File: packages/john/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+async function handleRequest(request) {
+  return new Response(html, {
+    headers: {
+      "content-type": "text/html;charset=UTF-8",
+    },
+  })
```
Comment (jbampton) [inline packages/john/index.js:34]:
```suggestion
  });
```

File: packages/john/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+addEventListener("fetch", event => {
+  return event.respondWith(handleRequest(event.request))
```
Comment (jbampton) [inline packages/john/index.js:38]:
```suggestion
  return event.respondWith(handleRequest(event.request));
```

File: packages/max/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const html = `<!doctype html>
+<body>
+  <h1>Hello Max The Super One!!!</h1>
+</body>`
```
Comment (jbampton) [inline packages/max/index.js:4]:
```suggestion
</body>`;
```

File: packages/max/index.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+async function handleRequest(request) {
+  return new Response(html, {
+    headers: {
+      "content-type": "text/html;charset=UTF-8",
+    },
+  })
```
Comment (jbampton) [inline packages/max/index.js:11]:
```suggestion
  });
```

File: packages/max/index.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+addEventListener("fetch", event => {
+  return event.respondWith(handleRequest(event.request))
```
Comment (jbampton) [inline packages/max/index.js:15]:
```suggestion
  return event.respondWith(handleRequest(event.request));
```

File: packages/umut/index.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+const html = `<!doctype html>
+<body>
+  <h1>Hello Umut The One!!</h1>
+</body>`
```
Comment (jbampton) [inline packages/umut/index.js:4]:
```suggestion
</body>`;
```

File: packages/umut/index.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+async function handleRequest(request) {
+  return new Response(html, {
+    headers: {
+      "content-type": "text/html;charset=UTF-8",
+    },
+  })
```
Comment (jbampton) [inline packages/umut/index.js:11]:
```suggestion
  });
```

File: packages/umut/index.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+addEventListener("fetch", event => {
+  return event.respondWith(handleRequest(event.request))
```
Comment (jbampton) [inline packages/umut/index.js:15]:
```suggestion
  return event.respondWith(handleRequest(event.request));
```

**Review by johnbampton — APPROVED** [review]

**Review by BaseMax — APPROVED** [review]
I am not sure why we need a packages directory and a personal profile for every person.
anyway LGTM.

**Comment by BaseMax** [comment]
Thanks

**Comment by BaseMax** [comment]
Merging is blocked
The base branch requires all conversations on code to be resolved.

**Comment by jbampton** [comment]
We won't have every person.  Just start with mine and yours.

**Pull request merged**